app/lib/project/MetapolatorProject.js

```diff
@@ -20,6 +20,7 @@ define([
   , 'yaml'
   , 'io/zipUtil'
   , 'io/InMemory'
+  , 'filesaver'
```

Comment:
You require it as a dependency, so you should receive it in the module closure function as an argument.

In this case you want to receive it under the name `saveAs`